product_analytic/demo/product_demo.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  © 2017 Akretion (Alexis de Lattre <alexis.delattre@akretion.com>)
```

Comment:
Please use "Copyright" instead of "C", as per https://github.com/OCA/maintainer-tools/tree/master/template/module/